VideoLocker/src/main/java/org/edx/mobile/model/api/CourseEntry.java

```diff
@@ -55,7 +57,7 @@ public void setStart(String start) {
     }
 
     public String getCourse_image(Config config) {
-        return config.getApiHostURL() + course_image;
```

Comment:
Hrrm. So my initial assumption was that since it could previously have returned null if both values were null (null + null is NullPointerException), the behavior shouldn't need to change, but in retrospect, making assumptions about this code doing something reasonable is probably foolish. Will investigate further. Good catch